api/admin/package.json

```diff
@@ -8,11 +8,7 @@
   },
   "author": "NYPL",
   "license": "Apache-2.0",
-  "engines": {
-    "node": ">=8.0.0",
-    "npm": ">=5.0.0"
-  },
   "dependencies": {
-    "simplified-circulation-web": "0.5.16"
+    "simplified-circulation-web": "0.6.0-rc"
```

Comment:
@keithbauer can you try running this with version `0.6.0-rc3`? I think it'll be the last "release candidate" before I make the final stable version.

Reply:
It seems to work for me.

Reply:
I just tried this out locally and it looks good to me on the pages that load without elasticsearch functionality.